house_rules/downtime.md

```diff
@@ -61,15 +61,20 @@ When performing downtime jobs, you can report your rolls, number of time units s
 
 Here are some example types of jobs that Guild members have performed, along with the corresponding skill rolls:
 * **Physical labor**: Athletics + Constitution
-* **Tavern cooking**: Dexterity + Wisdom
-* **Waiting tables**: Dexterity + Charisma
-* **Gardening**: Nature + Survival
+* **Tavern cooking**: Dexterity + Wisdom (#tavern)
```

Comment:
How about making the channel a header instead? That way it can be organized by activity type.

Reply:
Didn’t quite go full header but reordered and nested within the lost